unitary/alpha/quantum_world.py

```diff
@@ -146,3 +145,71 @@ def pop(
             self.post_selection[objects[idx]] = post_selection
 
         return results[0]
+
+    def get_histogram(self,
+                      objects: Optional[Sequence[QuantumObject]] = None,
+                      count: int = 100
+    ) -> List[Dict[int, int]]:
+        """Creates histogram based on measurements (peeks) carried out.
+
+        Parameters:
+            objects:    List of QuantumObjects
+            count:      Number of measurements
+
+        Returns:
+            A list with one element for each object. Each element contains a dictionary with
+            counts for each state of the given object.
+        """
+        if not objects:
+            objects = self.objects
+        peek_results = self.peek(objects=objects,
+                                 convert_to_enum=False,
+                                 count=count)
+        histogram = []
+        for obj in objects:
+            histogram.append({state: 0 for state in range(obj.num_states)})
+        for result in peek_results:
+            for idx in range(len(objects)):
+                histogram[idx][result[idx][0]] += 1
+        return histogram
+
+    def get_probabilities(self,
+                          objects : Optional[Sequence[QuantumObject]] = None,
+                          count: int = 100
+    ) -> List[Dict[int, float]]:
```

Comment:
Same here.  The result could be an Enum.

Reply:
In peek I used convert_to_enum=False, so here it should be int.  Later, maybe, we can write a nicer version, which can handle both enum or int, but this always converts the states to int.